package.json

```diff
@@ -46,48 +47,51 @@
     "@types/sinon": "^1.16.32",
     "@types/toastr": "^2.1.32",
     "@types/webpack-env": "^1.12.1",
+    "awesome-typescript-loader": "^3.1.3",
+    "babel-core": "^6.24.1",
+    "babel-preset-env": "^1.4.0",
     "chai": "^3.5.0",
-    "core-js": "^2.4.1",
-    "css-loader": "^0.25.0",
+    "css-loader": "^0.28.1",
     "deep-freeze": "0.0.1",
     "enzyme": "^2.6.0",
-    "es6-promise": "^4.0.5",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "file-loader": "^0.9.0",
-    "font-awesome": "^4.7.0",
+    "extract-text-webpack-plugin": "^2.1.0",
+    "file-loader": "^0.11.1",
     "html-webpack-plugin": "^2.22.0",
-    "istanbul-instrumenter-loader": "^1.0.0",
-    "json-loader": "^0.5.4",
+    "istanbul-instrumenter-loader": "^2.0.0",
     "karma": "^1.3.0",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.0.0",
     "karma-coverage": "^1.1.1",
+    "karma-coverage-istanbul-reporter": "^1.2.1",
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.1",
     "karma-sinon-chai": "1.2.4",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^1.8.0",
+    "karma-webpack": "^2.0.3",
     "mocha": "^3.1.2",
-    "node-sass": "^4.1.1",
+    "node-sass": "^4.5.2",
     "react-addons-test-utils": "^15.4.1",
-    "react-hot-loader": "^1.3.0",
+    "react-hot-loader": "next",
     "react-test-renderer": "^15.5.4",
     "redux-mock-store": "^1.2.1",
-    "sass-loader": "^4.1.1",
+    "sass-loader": "^6.0.3",
     "sinon": "^1.17.6",
     "sinon-chai": "^2.8.0",
-    "style-loader": "^0.13.1",
-    "ts-loader": "^0.8.2",
+    "style-loader": "^0.17.0",
     "tslint": "^4.2.0",
     "tslint-loader": "^3.3.0",
     "tslint-react": "^2.2.0",
-    "typescript": "^2.0.3",
-    "url-loader": "^0.5.7",
-    "webpack": "^1.13.1",
-    "webpack-dev-server": "^1.14.1"
+    "typescript": "2.2.2",
+    "url-loader": "^0.5.8",
+    "webpack": "^2.5.0",
+    "webpack-dev-server": "^2.4.5",
+    "webpack-merge": "^4.1.0"
   },
   "dependencies": {
     "bootstrap": "^3.3.7",
+    "core-js": "^2.4.1",
+    "es6-promise": "^4.0.5",
```

Comment:
I don't think so since `core-js` already has a `Promise` polyfill. I'll remove it.